### DISCUSSION TITLE
Inconsistent CPM data

### DISCUSSION THREAD

**Discussion by BertOnline** [apache/skywalking#9607] [Q&A] [answered]
Why are the values of CPM in the instance of the overview board inconsistent with those in the instance board.

![image](https://user-images.githubusercontent.com/17882970/190309565-65601463-2848-4301-a812-a6c516743041.png)
![image](https://user-images.githubusercontent.com/17882970/190309623-fec43d19-a180-4404-9227-1fa53b107eb0.png)

**Comment by wu-sheng** [comment, marked as answer]
Preview table list metrics are calculated by the UI. The other is group by in the database.

**Reply by wu-sheng** [reply]
There are no guaranteed consistent.

**Reply by BertOnline** [reply]
@wu-sheng Why does Service Instance Load multiply by 100 and then add up to equal the value of Service Load

![image](https://user-images.githubusercontent.com/17882970/190335133-f87ce5a7-1fc4-4ce1-82d8-bc8099cf1cee.png)

**Reply by wu-sheng** [reply]
This seems a UI bug. It applies a not existing formula.

**Reply by wu-sheng** [reply]
`"calculation": "percentage"` is mis-set on the dashboard setting.

**Reply by wu-sheng** [reply]
Fix through https://github.com/apache/skywalking/pull/9610
